app/Http/Requests/StartJoinMeeting.php

```diff
@@ -9,7 +9,8 @@ class StartJoinMeeting extends FormRequest
     public function rules()
     {
         return [
-            'name' => auth()->check() ? '' : 'required|min:2|regex:/^[a-zA-ZàáâäãåąčćęèéêëėįìíîïłńòóôöõøùúûüųūÿýżźñçčšžÀÁÂÄÃÅĄĆČĖĘÈÉÊËÌÍÎÏĮŁŃÒÓÔÖÕØÙÚÛÜŲŪŸÝŻŹÑßÇŒÆČŠŽ∂ð ,.\'-]+$/u|max:50',
+            'name'              => auth()->check() ? '' : 'required|min:2|regex:/^[a-zA-ZàáâäãåąčćęèéêëėįìíîïłńòóôöõøùúûüųūÿýżźñçčšžÀÁÂÄÃÅĄĆČĖĘÈÉÊËÌÍÎÏĮŁŃÒÓÔÖÕØÙÚÛÜŲŪŸÝŻŹÑßÇŒÆČŠŽ∂ð ,.\'-]+$/u|max:50',
+            'record_attendance' => 'required|in:not_accepted,accepted',
```

Comment:
Why not just a boolean?

Reply:
I'll replace it. I copied it from the file agreement and didn't change it. I will also replace it for the file download.